### ISSUE TITLE
TypeError: Cannot read properties of undefined

### ISSUE THREAD

**Issue by z6vba** [medusajs/medusa#1903]
### Describe the bug

```
$ medusa develop
Successfully compiled 0 files with Babel (7ms).
✔ Models initialized – 33ms
✔ Plugin models initialized – 7ms
✔ Repositories initialized – 66ms
✔ Database initialized – 123ms
✔ Strategies initialized – 12ms
✔ Services initialized – 144ms
✔ Express intialized – 10ms
⠋ Initializing pluginsTypeError: Cannot read properties of undefined (reading 'prototype')
    at node_modules/@medusajs/medusa/dist/loaders/plugins.js:340:54
    at step (node_modules/@medusajs/medusa/dist/loaders/plugins.js:33:23)
    at Object.next (node_modules/@medusajs/medusa/dist/loaders/plugins.js:14:53)
    at node_modules/@medusajs/medusa/dist/loaders/plugins.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (node_modules/@medusajs/medusa/dist/loaders/plugins.js:4:12)
    at node_modules/@medusajs/medusa/dist/loaders/plugins.js:332:87
    at Array.map (<anonymous>)
    at node_modules/@medusajs/medusa/dist/loaders/plugins.js:332:60
    at step (node_modules/@medusajs/medusa/dist/loaders/plugins.js:33:23)
⠦ Initializing plugins
```

### System information

Medusa version (including plugins): 1.3.1
Node.js version: v16.15.1
Database: postgres
Operating system: linux
Browser (if relevant):

### Steps to reproduce the behavior

1. medusa develop

**Comment by robin-multisafepay** [comment]
@z6vba did you manage to fix it?
And if so, how?

**Comment by z6vba** [comment]
@robin-multisafepay I fixed it at pr #1948
